src/Module.zig

```diff
@@ -845,6 +845,9 @@ pub const File = struct {
     /// successful, this field is unloaded.
     prev_zir: ?*Zir = null,
 
+    /// Whether the file is Zig or ZON.
+    mode: Ast.Mode,
```

Comment:
I assume this is only populated when `status == .success_zir` or similar - could you document that?